src/components/common/BlanketDragZone/BlanketDragZone.js

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+
+import blanket from "../../../utils/blanket/index.js";
+import loadingAnimationImage from "../../../images/loading-animation.gif";
+
+export default function BlanketDragZone({ handleDrop }) {
+  return (
+    <div
+      style={{
+        position: "fixed",
+        left: 0,
+        top: 0,
+        width: "100vw",
+        height: "100vh",
+        display: "flex",
+        justifyContent: "center",
+        alignItems: "center",
+        pointerEvents: "none",
+        zIndex: 100000000,
+      }}
+    >
+      <input
+        style={{
+          width: "300vw",
```

Comment:
wouldn't anything above 100vw be unseen/unnecessary?

Reply:
I did this a while back, but if I remember correctly it was just me being cautious to make sure it covered the entire screen with room to spare. I *think* there may have been an issue where one of the corners wasn't working with 100%, so I just cranked everything up to 300